src/App.vue

```diff
@@ -36,6 +42,12 @@ export default createComponent({
         event: "error"
     },
     setup(props: {}, context: SetupContext) {
+        onMounted(() => {
+            // prevent going to a 404 when launching as an Electron app
+            // see https://nklayman.github.io/vue-cli-plugin-electron-builder/guide/commonIssues.html#blank-screen-on-builds-but-works-fine-on-serve
+            router.push("/");
```

Comment:
There must be something else we can do here. This is doing some wonky stuff to the web build. Maybe `if` it out? Maybe `replace` instead of `push` too?